### PR TITLE
[nl server] don't try to read vertex ai model info if custom dc

### PR DESCRIPTION
**Pull request by chejennifer**

### CODE REVIEW

File: nl_server/config.py [hunk 1]
```diff
@@ -112,6 +113,10 @@ class EmbeddingsConfig:
 # Get Dict of vertex ai model to its info
 #
 def _get_vertex_ai_model_info() -> Dict[str, any]:
+  # Custom DC doesn't use vertex ai so just return an empty dict
+  if is_custom_dc():
```
Comment (keyurva) [inline nl_server/config.py:117]: Do we want custom dcs to use them?

If we do, the model info can be added in the custom dc docker image [here](https://github.com/datacommonsorg/website/blob/63b5002f1c1a0b9dbbdca0ad1a0bb3c66801136e/build/web_compose/Dockerfile#L67).

Comment (chejennifer) [reply nl_server/config.py:117]: hm, I don't think we want custom dcs to use them right now

Comment (keyurva) [reply nl_server/config.py:117]: SG. Consider adding a TODO if we plan to in the future.

**Review by keyurva — APPROVED** [review]
Thanks for the fix!

File: nl_server/config.py (hunk 1 above)
Comment (chejennifer) [reply nl_server/config.py:119]: done

**Pull request merged**